controller/array_action/array_mediator_ds8k.py

```diff
@@ -29,9 +30,9 @@
 ERROR_CODE_ALREADY_FLASHCOPY = '000000AE'
 ERROR_CODE_VOLUME_NOT_FOUND_OR_ALREADY_PART_OF_CS_RELATIONSHIP = '00000013'
 
-FLASHCOPY_PERSISTENT_OPTION = "persistent"
-FLASHCOPY_NO_BACKGROUND_COPY_OPTION = "no_background_copy"
-FLASHCOPY_PERMIT_SPACE_EFFICIENT_TARGET = "permit_space_efficient_target"
+FLASHCOPY_PERSISTENT_OPTION = ds8k_types.DS8K_OPTION_PER
```

Comment:
btw, I know it's not really part of this ticket, but why is `FLASHCOPY_PERSISTENT_OPTION` **always** added to `options` outside `_create_flashcopy`, but `FLASHCOPY_PERMIT_SPACE_EFFICIENT_TARGET` is added inside?

Reply:
because we always want to permit ESE but not necessarily always want the flashcopy to be persistent. now we do, but more functions will e added and that not always will be the case

Reply:
when will we not want the flashcopy to be persistent?

Reply:
when we wont be needing the source, like in clone..

Reply:
why wouldn't we want to know the clone source?
does it matter if the source is a snapshot or a volume?

Reply:
I dont know what will want to know right now, just tried to not be too specific..

Reply:
so, since we currently don't have any solid reason to believe otherwise, I think that as long as the code requires both parameters everywhere, then it should be written accordingly (to add both parameters inside `_create_flashcopy`).
and if we ever want to change it, we can always do.